packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -122,13 +158,30 @@ The default `HistoryQuery` component that is used in every suggestion can be rep
 Remember that if HistoryQuery component wasn't used the `handleHistoryQuerySelection` method needs
 to be implemented emitting the needed events.
 
-```vue
-<HistoryQueries>
-  <template #suggestion="{ suggestion }">
-    <img class="x-history-query__icon" src="./history-query-extra-icon.svg"/>
-    <HistoryQuery :suggestion="suggestion"/>
-  </template>
-</HistoryQueries>
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work